docs/modules/ROOT/pages/reference/framework-requirements.adoc

```diff
@@ -0,0 +1,46 @@
+= Framework Requirements
+
+As discussed on xref:explanations/what_is.adoc["What is it?"], a *framework* for self-service provisioning of services out of a Kubernetes cluster is a core part of the application catalog.
+
+This page discusses requirements the framework must adhere to:
+
+Reusability::
+Base parts of the framework should be reusable by the Open Source community and VSHNeers.
```

Comment:
From a business perspective: Why does the framework need to be reusable by the Open source community? The open source community is unlikely going to help the VSHN-opinionated framework. (I'm not saying it couldn't be Open source, but making it _reusable_ adds more complexity in various places). Similar to APPUiO Cloud, where we thought we apply the resellable aspects, but it's not widely used out in the open and still very VSHN-opinionated.

Reply:
With that part I meant when we use Crossplane that we write a "base part" provider-cloudscale which is reusable. How should I reformulate that part?

Again compare with Puppet: When I needed to install and configure Apache I wrote a reusable base module for Apache and an opinionated non-reusable profile.